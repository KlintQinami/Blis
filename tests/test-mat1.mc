int main() {
    vec3 tom = vec3(1337., 42., 300.);
    vec3 jeff = vec3(1., 2., 3.);
    vec3 john = vec3(4., 5., 6.);
    mat3x3 bob = mat3x3(tom, jeff, john);
    printf(bob.x.x);
    printf(bob.x.y);
    printf(bob.x.z);
    printf(bob.y.x);
    printf(bob.y.y);
    printf(bob.y.z);
    printf(bob.z.x);
    printf(bob.z.y);
    printf(bob.z.z);
    return 0;
}
