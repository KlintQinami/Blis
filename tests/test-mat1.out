1337.000000
42.000000
300.000000
1.000000
2.000000
3.000000
4.000000
5.000000
6.000000
